config/api.yml

```diff
@@ -301,6 +301,10 @@
         :identifier: miq_request_approval
       - :name: edit
         :identifier: miq_request_edit
+    :request_tasks_subcollection_actions:
+      :post:
+      - :name: edit
+        :identifier: miq_request_edit
```

Comment:
If all these use the same identifier, they can be consolidated under `:request_tasks`: 

```yaml
:request_tasks:
  :subcollection_actions:
    :post:
    - :name: edit
      :identifier: miq_request_edit
```